frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { useEffect, useState } from "react";
+import { Spinner } from "react-bootstrap";
```

Comment:
The logic here with React's state management looks great, well done! One small thing I thought of is maybe using React Bootstrap's [Placeholders](https://react-bootstrap.github.io/components/placeholder/) instead of a spinner might look better? This way there's no dramatic content shift and the user can anticipate what the area will look like. What do you think would look better, as this isn't necessarily a change but a discussion!

Reply:
Ooo I like the look of it. I'll try to add it.

Reply:
Actually it seems like Placeholder is not in the version of React Bootstrap in the frontend container. We are in version 1.x and Placeholder is in 2.x.

Reply:
hmm, @AstridKery do you think we could/should upgrade React Bootstrap? Or just stick with what we have...?

Reply:
Personally though, I like the spinner :)

Reply:
Ok sweet, sounds good with me then.